lib/iris/_lazy_data.py

```diff
@@ -95,6 +95,10 @@ def as_concrete_data(data, **kwargs):
     Returns:
         A NumPy `ndarray` or masked array.
 
+    .. note::
+        Specific dask options for computation are controlled by
+        :class:`iris.options.Parallel`.
```

Comment:
@bjlittle it should be `:class:iris.config.Parallel` as `Parallel` is (a) actually a class that can be linked to by Sphinx, and (b) has a docstring. I'll change this, and all of the similar below...